src/launch/policy.h

```diff
@@ -114,6 +115,7 @@ struct Policy {
                 .gid_tree = C_RBTREE_INIT,                                              \
                 .selinux_list = C_LIST_INIT((_x).selinux_list),                         \
                 .apparmor_mode = CONFIG_APPARMOR_ENABLED,                               \
+                .bus_type = CONFIG_BUS_TYPE_SYSTEM,                                     \
```

Comment:
Is `system` actually the default? The man-page doesn't mention it, and I didn't check the dbus sources. Did you?

Reply:
No, I did not check when writing this and totally forgot about it. I just did check dbus-daemon. Apparently it supports arbitrary bus types incl. the bus type being empty and then has special handling for "session" and "system". So parsing needs to go and instead the string should be used directly.

Reply:
Yeah, I think we need to avoid parsing it and just send it through, indeed.